legacy/build-deploy-docker-compose.sh

```diff
@@ -206,8 +215,8 @@ You can run docker compose config locally to check that your docker-compose file
 ##############################################"
   exit 1
 fi
+beginBuildStep "Initial Environment Setup"
```

Comment:
I would probably move this one to line 146 instead of having `beginBuildStep "Build Preparation"` there.

Reply:
Thanks! Will update it now